float32 delay_before_processing

uint8 set_pressure
bool should_tamp
bool should_fire

